pyccel/codegen/printing/ccode.py

```diff
@@ -1424,6 +1439,9 @@ def _print_Comment(self, expr):
 
         return '/*' + comments + '*/'
 
+    def _print_PyccelSymbol(self, expr):
+        return expr
```

Comment:
Why is this added, but `_print_Symbol` not removed? Is this a new requirement? Do you know where it comes from?

Reply:
functions names are PyccelSymbol